tools/generator/Tests-Core/expected.ji/Android.Text.SpanTypes.cs

```diff
@@ -1,5 +1,6 @@
 namespace Android.Text {
   public enum SpanTypes {
+    [global::Android.Runtime.IntDefinition (null, JniField = "I:android/text/Spanned.SPAN_COMPOSING")]
```

Comment:
WTF. Those "I:" are actually from map.csv to indicate that the type is an interface (not a class), which should not have been required from the beginning. I made a change that kills those "I:" prefixes.